### ISSUE TITLE
I can't run run_sr.lua

### ISSUE THREAD

**Issue by FIGHTFORGLO** [leehomyc/Photo-Realistic-Super-Resoluton#15]
The following is my mistake：

 t_folder=dataset/ model_folder=model/ th run_sr.lua                                                                          {
  ntrain : inf
  beta1 : 0.9
  name : "super_resolution"
  dataset : "folder"
  niter : 250
  lr : 0.001
  model_folder : "model/"
  gpu : 1
  nThreads : 4
  t_folder : "dataset/"
  batchSize : 32
  loadSize : 96
}
Starting donkey with id: 1 seed: 1
table: 0x40c0e918
Starting donkey with id: 3 seed: 3
table: 0x41c70418
Starting donkey with id: 4 seed: 4
table: 0x410d8418
Starting donkey with id: 2 seed: 2
table: 0x41b53db0
Creating train metadata
table: 0x41632658
Creating train metadata
table: 0x410d8a28
Creating train metadata
table: 0x40222228
Creating train metadata
table: 0x40c0ebf0
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
load the large concatenated list of sample paths to self.imagePath
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
/root/torch/install/bin/luajit: /root/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 4 callback] .../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:210: Could not find any image file in the given input paths
stack traceback:
        [C]: in function 'assert'
        .../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:210: in function '__init'
        /root/torch/install/share/lua/5.1/torch/init.lua:91: in function </root/torch/install/share/lua/5.1/torch/init.lua:87>
        [C]: in function 'dataLoader'
        ...tic-Super-Resoluton-master/data/donkey_folder_supres.lua:69: in main chunk
        [C]: in function 'dofile'
        /home/Photo-Realistic-Super-Resoluton-master/data/data.lua:33: in function </home/Photo-Realistic-Super-Resoluton-master/data/data.lua:23>
        [C]: in function 'xpcall'
        /root/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
        /root/torch/install/share/lua/5.1/threads/queue.lua:65: in function </root/torch/install/share/lua/5.1/threads/queue.lua:41>
        [C]: in function 'pcall'
        /root/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
        [string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
        [C]: in function 'error'
        /root/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
        /root/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
        /root/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
        /root/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
        /home/Photo-Realistic-Super-Resoluton-master/data/data.lua:21: in function 'new'
        run_sr.lua:31: in main chunk
        [C]: in function 'dofile'
        /root/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
        [C]: at 0x00406670

I'm sure I have the training data I need in my catalog.
Can you help me?

**Comment by why21** [comment]
I have the same problem. Have you solved it?
